static/app/views/releases/detail/activity/types.tsx

```diff
@@ -0,0 +1,39 @@
+import type {Group} from 'sentry/types';
+
+export enum ReleaseActivityType {
+  CREATED = 'created',
+  DEPLOYED = 'deployed',
+  FINISHED = 'finished',
+  ISSUE = 'issue',
```

Comment:
I made a change to the BE to have the enum values be all caps, could we update these enum values to be all caps as well.